FFXIVClientStructs/FFXIV/Client/UI/Misc/RaptureTextModule.cs

```diff
@@ -54,17 +54,106 @@ public unsafe partial struct RaptureTextModule {
+    public partial CStringPointer FormatAddonText1IntInt(uint addonId, int intParam1, int intParam2);
+
+    [MemberFunction("E8 ?? ?? ?? ?? 8B 7D FF 45 33 FF")]
     public partial CStringPointer FormatAddonText1IntIntUInt(uint addonId, int intParam1, int intParam2, uint uintParam);
 
-    [MemberFunction("E8 ?? ?? ?? ?? 41 8D 55 0B")] // FormatAddonText2<int>
-    public partial CStringPointer FormatAddonText2Int(uint addonId, int value);
+    [MemberFunction("E8 ?? ?? ?? ?? EB 38 49 8B D2")]
+    public partial CStringPointer FormatAddonText1IntIntUIntUInt(uint addonId, int intParam1, int intParam2, uint uintParam1, uint uintParam2);
+
+    [MemberFunction("E8 ?? ?? ?? ?? 8B 5C 24 44 48 8B D0")]
+    public partial CStringPointer FormatAddonText1IntString(uint addonId, int intParam, CStringPointer strParam);
+
+    // FormatAddonText2
+    [MemberFunction("E8 ?? ?? ?? ?? 4C 8B C5 48 89 44 24 ??")]
+    public partial CStringPointer FormatAddonText2String(uint addonId, CStringPointer strParam);
+
+    [MemberFunction("E8 ?? ?? ?? ?? EB 67 48 8B 7E 10")]
+    public partial CStringPointer FormatAddonText2StringInt(uint addonId, CStringPointer strParam, int intParam);
+
+    [MemberFunction("E8 ?? ?? ?? ?? 48 8B 7C 24 ?? EB 14")]
+    public partial CStringPointer FormatAddonText2StringIntUInt(uint addonId, CStringPointer strParam, int intParam, uint uintParam);
+
+    [MemberFunction("40 55 53 56 57 41 54 41 56 41 57 48 8B EC 48 83 EC 50 48 8B 05 ?? ?? ?? ?? 48 33 C4 48 89 45 F8 48 8D B9 ?? ?? ?? ?? 48 8B F1 48 8B CF 45 8B F9 49 8B D8 44 8B E2 E8 ?? ?? ?? ?? 48 85 DB 48 8D 15 ?? ?? ?? ?? 48 8D 8E ?? ?? ?? ?? 48 0F 45 D3 E8 ?? ?? ?? ?? 33 D2 45 33 C0 8D 4A 70 E8 ?? ?? ?? ?? 48 8B D8 48 85 C0 74 18 48 8D 96 ?? ?? ?? ?? 48 8B C8 E8 ?? ?? ?? ?? C7 43 ?? ?? ?? ?? ?? EB 02 33 DB 48 8D 45 E0 48 89 5D E0 48 8D 55 E0 48 89 45 E8 48 8B CF C6 45 F0 01 E8 ?? ?? ?? ?? 48 8B 5D E8 48 85 DB 74 27 80 7D F0 01 75 21 48 8B 1B FF 4B 68 83 7B 68 00 7F 15 48 8B CB E8 ?? ?? ?? ?? BA ?? ?? ?? ?? 48 8B CB E8 ?? ?? ?? ?? 33 C0 C6 45 F0 00 48 89 45 E0 48 8D 55 E0 48 8D 45 E0 44 89 7D E0 48 8B CF 48 89 45 E8 E8 ?? ?? ?? ?? 48 8B 5D E8 48 85 DB 74 27 80 7D F0 01 75 21 48 8B 1B FF 4B 68 83 7B 68 00 7F 15 48 8B CB E8 ?? ?? ?? ?? BA ?? ?? ?? ?? 48 8B CB E8 ?? ?? ?? ?? 33 C0 C6 45 F0 00 48 89 45 E0 48 8D 55 E0 8B 45 60 48 8B CF 89 45 E0 48 8D 45 E0 48 89 45 E8 E8 ?? ?? ?? ?? 48 8B 5D E8 48 85 DB 74 27 80 7D F0 01 75 21 48 8B 1B FF 4B 68 83 7B 68 00 7F 15 48 8B CB E8 ?? ?? ?? ?? BA ?? ?? ?? ?? 48 8B CB E8 ?? ?? ?? ?? 33 C0 C6 45 F0 00 48 89 45 E0 48 8D 55 E0 8B 45 68 48 8B CF 89 45 E0 48 8D 45 E0 48 89 45 E8 E8 ?? ?? ?? ?? 48 8B 5D E8 48 85 DB 74 27 80 7D F0 01 75 21 48 8B 1B FF 4B 68 83 7B 68 00 7F 15 48 8B CB E8 ?? ?? ?? ?? BA ?? ?? ?? ?? 48 8B CB E8 ?? ?? ?? ?? 4C 8D 86 ?? ?? ?? ??")]
+    public partial CStringPointer FormatAddonText2StringIntUIntUInt(uint addonId, CStringPointer strParam, int intParam, uint uintParam1, uint uintParam2);
```

Comment:
```suggestion
```
No xrefs and a giant signature that detects multiple functions when scanning in IDA